guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -2,12 +2,12 @@
 
 = Upgrading {SmartProxyServer}s
 
-This section describes how to upgrade {SmartProxyServer}s from {ProductVersionPrevious} to {ProductVersion}.
+This section describes how to upgrade {SmartProxyServer}s from {ProjectVersionPrevious} to {ProjectVersion}.
 
 .Before You Begin
 
 * You must upgrade {ProjectServer} before you can upgrade any {SmartProxyServer}s. Note that you can upgrade {SmartProxies} separately from {Project}. For more information, see xref:upgrading_capsules-separately-from-satellite_upgrade-guide[].
-* Ensure the {ProjectName} {SmartProxy} {ProductVersion} repository is enabled in {ProjectServer} and synchronized.
+* Ensure the {ProjectName} {SmartProxy} {ProjectVersion} repository is enabled in {ProjectServer} and synchronized.
 * Ensure that you synchronize the required repositories on {ProjectServer}. For more information, see xref:synchronizing_the_new_repositories[].
```

Comment:
The xref here doesn't work because this section has been filtered out of the Katello guide. Do we need it? If so, add it in, if not, wrap it in a Satellite ifdef